cmd/manager/main.go

```diff
@@ -107,10 +114,32 @@ func main() {
 		os.Exit(1)
 	}
 
-	// Create Service object to expose the metrics port.
-	_, err = metrics.ExposeMetricsPort(ctx, metricsPort)
+	if err = serveCRMetrics(cfg); err != nil {
```

Comment:
Feels bad to have metrics that we never even tried. But it is not a regression compared to the original code, just complaining.

Reply:
Let's try to simplify not used stuff at follow-up PRs